### DISCUSSION TITLE
Content Management System that uses Bootstrap CSS

### DISCUSSION THREAD

**Discussion by jesperby** [twbs/bootstrap#39366] [Q&A]
Is there a CMS that uses Bootstrap CSS? I'm thinking of a CMS where you, inside the CMS settings, point to the Bootstrap CSS file, and the CMS then uses it 100% (no graphic design settings stored within the CMS).

**Comment by patkon** [comment]
I think, most of the CMS using Template engines like smarty or blade. So you can use the css framework you want.
But maybe I'm not understanding the question correctly. what exactly do you want to achieve?

**Reply by jesperby** [reply]
I'm looking for this:
<img width="751" alt="Skärmavbild 2023-11-07 kl  12 53 03" src="https://github.com/twbs/bootstrap/assets/28442069/a567da04-a917-43e1-a2ea-3e04e8ac175c">

**Reply by Jonwebiny** [reply]
Webiny is a open-source headless CMS solution. You can manage global CSS styles from within the CMS and then use any SSG of your choice. I'm not sure exactly what you are looking for from your architecture diagram, but perhaps another way of achieving the same goal. Please review Global Styles within https://www.webiny.com/docs/page-builder/theming/best-practices
If you want to discuss further you can join the Webiny slack community and reach me there or e-mail me on John@Webiny.com
